frontend/src/components/Answer/Answer.tsx

```diff
@@ -74,7 +74,17 @@ export const Answer = ({
             </Stack.Item>
 
             <Stack.Item grow className={styles.markdownContent}>
-                <ReactMarkdown remarkPlugins={[remarkGfm]} rehypePlugins={[rehypeRaw]}>
+                <ReactMarkdown
+                    remarkPlugins={[remarkGfm]}
+                    rehypePlugins={[rehypeRaw]}
+                    components={{
+                        a: ({ node, ...props }) => (
+                            <a {...props} target="_blank" rel="noopener noreferrer" style={{ color: "#85a717", textDecoration: "none" }}>
```

Comment:
@ramirezmorac2 shouldn't be this a <link component for react router instead of a element?